modules/startup_sql_queries.js

```diff
@@ -177,6 +177,17 @@ module.exports = async function (databasePool) {
     );"
   );
 
+  await databasePool.query(
+    "CREATE TABLE IF NOT EXISTS forms (\
+      form_id varchar(64),\
```

Comment:
formName max lenght is 64. Considering IDs are formName + serverId, this isn't large enough to fit forms IDs. We must increase this size!